setup.py

```diff
@@ -26,7 +26,6 @@
           'cryptography>=0.5',
           'cython>=0.21.1',
           'dnsq>=1.1.6',
-          'expiringdict>=1.1.2',
```

Comment:
Well, it is a transitive dependency - flanker->dnsq->expiringdict, but you are right it is not needed here.